swig/swigResqml2Include.i

```diff
@@ -1855,6 +1862,14 @@ namespace gsoap_eml2_3
 	};
 }
 
+namespace std {
+	%template(StratigraphicOccurrenceInterpretationVector) vector<RESQML2_NS::StratigraphicOccurrenceInterpretation *>;
```

Comment:
I think it is not possible (not recommended) to define some vectors of abstract classes because SWIG would not instantiate the right child classes in vectors.
That's the reason of all the getters (count and item) for each data object.

Reply:
Done.